src/Domain/Location.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Shouze\ParkedLife\Domain;
+
+class Location
+{
+    private $latitude;
+
+    private $longitude;
+
+    public function __construct($latitude, $longitude)
+    {
+        $this->latitude = $latitude;
+        $this->longitude = $longitude;
+    }
+
+    public static function fromString(string $locationAsString)
+    {
+        list($latitude, $longitude) = explode(',', $locationAsString);
+
+        return new Location($latitude, $longitude);
```

Comment:
because I find that more readable as far as the class is final. That is not the case for now :p